app/assets/stylesheets/mixins/course/forum/_forum_post.scss

```diff
@@ -0,0 +1,31 @@
+@mixin forum-post {
+  .user {
+    float: left;
+    margin-right: 1em;
+    text-align: right;
+    width: 7em;
+  }
+
+  .contents {
+    border: 1px solid #ddd;
+    border-radius: 4px;
+    box-shadow: 0 1px 1px rgba(0, 0, 0, 0.075);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.075)` should only be used in variable declarations; they should be referred to via variable everywhere else.